generate.coffee

```diff
@@ -107,6 +107,7 @@ generateOrders = (total, customers, addresses, products) ->
       entity_id: index
       items: items
       grand_total: grandTotal
+      base_grand_total: grandTotal
```

Comment:
Could you also add a `base_discount_amount` column? You can add it in a separate PR if you want, but this looks like its small enough, that you could do it here.

This field's data type should be `DECIMAL(12,4)`.